website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -45,6 +45,10 @@ git push origin 0.69-stable
 
 Before continuing further, follow the [testing guide](/contributing/release-testing) to ensure the code doesn't have any major issues.
 
+:::info
+Since testing is a time consuming activity (>1 hrs) it is recommended that the release crew coordinates on the steps above then do testing on at least two separate systems in an async way.
+:::
+
```

Comment:
What about creating a shared snippet and reusing it here and in `website/contributing/release-candidate-patch.md` and in `website/contributing/release-stable-patch.md`?

Reply:
is there any example of such thing anywhere else on the website?

Reply:
yup, [this](https://github.com/facebook/react-native-website/blob/main/docs/_markdown-new-architecture-warning.mdx?plain=1) is the shared snippet and [this](https://github.com/facebook/react-native-website/blob/main/docs/new-architecture-app-modules-ios.md?plain=1#L6-L9) is how it is used.

Reply:
ok, thanks for the reference - done!